fileversion.go

```diff
@@ -253,25 +265,47 @@ func (self *FileVersion) GetFileInfoTranslations() ([]FileInfoTranslation, error
 		return nil, NewWindowsError("VerQueryValue", err)
 	}
 
-	result := make([]FileInfoTranslation, 0)
+	result := make([]FileVersionTranslation, 0)
 	if len == 0 {
 		return result, nil
 	}
 
 	ti := fit
-	for i := 0; i < int(len/uint32(unsafe.Sizeof(*ti))); i++ {
-		result = append(result, FileInfoTranslation{Language: ti.Language, CodePage: ti.CodePage})
-		ti = (*FileInfoTranslation)(unsafe.Pointer(uintptr(unsafe.Pointer(ti)) + unsafe.Sizeof(*ti)))
+	l := int(len / uint32(unsafe.Sizeof(*ti)))
+	for i := 0; i < l; i++ {
+		result = append(result, FileVersionTranslation{Language: uint(ti.Language), CodePage: uint(ti.CodePage)})
+		ti = (*fileVersionTranslation)(unsafe.Pointer(uintptr(unsafe.Pointer(ti)) + unsafe.Sizeof(*ti)))
 	}
 	return result, nil
 }
 
-func (self *FileVersion) GetStringFileInfo(translation FileInfoTranslation, stringName string) (string, error) {
+func (self *FileVersion) GetStringFileInfo(translation FileVersionTranslation) (*StringFileInfo, error) {
+	tr, err := self.GetTranslations()
```

Comment:
No need to call `GetTranslations` here.  Just build the `StringFileInfo` and return it.  If there's no data for the specified language, it'll just fail out of `VerQueryValue` later on.

Also, can just return a single value without the error, since there will be no opportunity for this function to fail.